### ISSUE TITLE
question regarding "def _average_weights"

### ISSUE THREAD

**Issue by chrisbeekman** [simon-larsson/keras-swa#9]
Hi!
Very nice project. 
I do have a question though regarding the function where the running swa weights average is being calculated. 
In "def _average_weights()", you have, 

return [(swa_w * (epoch - self.start_epoch) + w)
                / ((epoch - self.start_epoch) + 1)
                for swa_w, w in zip(self.swa_weights, self.model.get_weights())]

Shouldn't the weighting be done in multiples of the swa_freq, in case you use a cyclic learning rate? So instead of 
(epoch - self.start_epoch) --> (epoch - self.start_epoch) / self.swa_freq (we only average every self.swa_freq epochs, right?!)
Otherwise you over-emphasize the importance of the weights at the beginning of the swa run (also see algorithm 1 of the Izmailov paper). Or am I missing something?

Best, Chris

**Comment by simon-larsson** [comment]
No, I think you are correct. I will look into it this weekend. Nice catch! :)

**Comment by simon-larsson** [comment]
Added in version 1.6
